policybot/pkg/storage/types.go

```diff
@@ -356,3 +356,42 @@ type UserAffiliation struct {
 	StartTime    time.Time
 	EndTime      time.Time
 }
+
+type AllBaseSha struct {
+	BaseSha string
+}
+
+type LatestBaseSha struct {
+	BaseSha        string
+	LastFinishTime time.Time
+	NumberofTest   int64
+}
+
+type PostSubmitTestResultDenormalized struct {
```

Comment:
Are all of these fields necessary to support the dashboard?  When we were using data studio, we took the kitchen sink approach to columns, because we weren't sure what would be useful.  Now that our usecase has tightened, can we limit ourselves to columns we are using?